### DISCUSSION TITLE
Card and Grid props

### DISCUSSION THREAD

**Discussion by pradyuk** [twilio-labs/paste#879] [Q&A] [answered]
Hi - I am trying to edit the "min height" prop for the card and grid components and "align-items" for grid. [This](https://code.hq.twilio.com/twilio/console/pull/new/CON-5960
) is the PR I am working on and what I am trying to do is to make sure the cards in the same row have the same height, and the height is based on the card with largest amount of content. We are trying to make the card size fit to the content. Can you please suggest how I can go about it?

**Comment by TheSisb** [comment]
Hi @pradyuk - I've added a ticket to provide align-items to Grid. I think that will be sufficient to achieve the desired UI.  We'll prioritize it into an upcoming sprint and I'll get back to you when the fix is in. Thanks!

**Reply by pradyuk** [reply]
@TheSisb Thanks fo that. Can we also get padding the in the buttons?

**Reply by TheSisb** [reply]
Sorry, I don't understand your question.  You want padding props on the Button component?  Can you share more about your use-case please?

**Comment by SiTaggart** [comment, marked as answer]
Hi @pradyuk 

A fairly basic but effectively solution for equal height columns in a row just got added to the Grid component. https://paste.twilio.design/layout/grid#equal-column-heights

It's not quite the perfect "it just works" solution, as we can only control the grid and column behavior. We don't have much control over the content you provide it, but we've tried our best to make something that will work pretty effectively most of the time.

Let us know what you think.
